src/openrct2/ride/Ride.cpp

```diff
@@ -7287,19 +7288,21 @@ bool ride_has_adjacent_station(Ride* ride)
             /* Get the map element for the station start. */
             auto stationZ = ride->stations[stationNum].Height;
```

Comment:
actually, do 
```c++
auto stationStart = CoordsXYZ{ ride->stations[stationNum].Start, ride->stations[stationNum].Height }
```

The `isNull` check will work just as well and all of the usages below will just use `stationStart` directly

Reply:
And the type of stationStart also is more obvious like this...